content/en/docs/ops/configuration/traffic-management/protocol-selection/index.md

```diff
@@ -12,9 +12,34 @@ owner: istio/wg-networking-maintainers
 test: no
 ---
 
-Istio supports proxying all TCP traffic by default, but in order to provide additional capabilities,
-such as routing and rich metrics, the protocol must be determined.
-This can be done automatically or explicitly specified.
+Istio supports proxying any TCP traffic. This includes HTTP, HTTPS, gRPC, as well as raw TCP protocols.
+In order to provide additional capabilities, such as routing and rich metrics, the protocol must be determined. This can be done automatically or explicitly specified.
+
+Non-TCP based protocols, such as UDP, are not proxied. These protocols will continue to function as normal, without
+any interception by the Istio proxy. They are not supported at gateways.
+
+## Automatic protocol selection
+
+Istio can automatically detect HTTP and HTTP/2 traffic. If the protocol cannot automatically be determined, traffic will be treated as plain TCP traffic.
+
+### Server first protocols
+
+Some protocols are "Server First" protocols, which means the server will send the first bytes. In these cases, automatic protocol detection
+will not function correctly, as Istio will determine the protocol based on the initial bytes. If you are using a server first protocol, you should
+explicitly declare the protocol following the instructions below.
```

Comment:
```suggestion
follow the [explicit protocol selection instructions](#explicit-protocol-selection) to declare the protocol.
```

Reply:
Should it be `#manual-protocol-selection` or do I not understand how this works?

Reply:
I thought renaming the section (manual -> explicit) would be better, since the text always uses the word explicit instead of manual. But that's just my preference.

Reply:
I agree